app/controllers/api_controller.rb

```diff
@@ -1,5 +1,6 @@
 class APIController < ActionController::Base
```

Comment:
```suggestion
class APIController < ActionController::API
```

à la place sinon

Reply:
Ca doit être iso, as you wish

Reply:
ça passait pas les tests c'est pour ça que je proposais ça

Reply:
Yes, cf le deuxième change : le double skip d'une même action lève une erreur. 

Là c'est vert, je merge / deploy.